HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -76,6 +76,25 @@ def self.apply_building_construction(hpxml, cfa, nbeds)
       hpxml.building_construction.conditioned_building_volume = cfa * hpxml.building_construction.average_ceiling_height
     end
     hpxml.building_construction.number_of_bathrooms = Float(Waterheater.get_default_num_bathrooms(nbeds)).to_i if hpxml.building_construction.number_of_bathrooms.nil?
+    if hpxml.building_construction.has_flue_or_chimney.nil?
+      hpxml.building_construction.has_flue_or_chimney = false
+      hpxml.heating_systems.each do |heating_system|
+        if [HPXML::HVACTypeFurnace, HPXML::HVACTypeBoiler].include? heating_system.heating_system_type
```

Comment:
Suggest adding additional heating types: WallFurnace, FloorFurnace, Stove, and FixedHeater. Note that you'll need to handle AFUE for some and Percent efficiency for others.